lib/embulk/input/mixpanel.rb

```diff
@@ -159,13 +160,13 @@ def run
         prev_latest_fetched_time = task[:latest_fetched_time] || 0
         prev_latest_fetched_time_format = Time.at(prev_latest_fetched_time).strftime("%F %T %z")
         current_latest_fetched_time = prev_latest_fetched_time
-        @dates.each_slice(task[:slice_range]) do |sliced_dates|
+        @dates.each_slice(task[:slice_range]) do |dates|
```

Comment:
Using both of @dates and `dates` in same block is confusing.

Reply:
I will change it